package.json

```diff
@@ -1,61 +1,47 @@
 {
   "name": "svelte-seo",
-  "version": "1.4.1",
-  "svelte": "src/SvelteSeo.svelte",
-  "module": "dist/index.min.mjs",
-  "main": "dist/index.min.js",
-  "types": "types/index.d.ts",
-  "repository": "git@github.com:artiebits/svelte-seo.git",
+  "version": "0.0.1",
   "license": "MIT",
+  "repository": "git@github.com:artiebits/svelte-seo.git",
   "scripts": {
-    "build": "rollup -c",
-    "dev": "rollup -c rollup.config.dev.js -w",
-    "start": "sirv dev",
-    "cy:run": "cypress run",
-    "cy:open": "cypress open",
-    "test": "run-p --race dev cy:run",
-    "prepublishOnly": "yarn build"
```

Comment:
please return this line back, needed it to publish a new version of the package

Reply:
but it doesn't use cypress to run the tests. Forgotten??

Reply:
I am talking about `"prepublishOnly": "yarn build"` :)